### ISSUE TITLE
Batchsize and lr of Swin-l+HTC

### ISSUE THREAD

**Issue by BinZhu-ece** [open-mmlab/mmdetection#6356]
Batchsize and lr of Swin-l+HTC

Thank you for your outstanding work, I only have 8 A-100cards, can I train  batchsize=1 each card and adjust the learning rate to 0.0001 / 2? Is there a negative impact on this relatively large model accuracy of HTC+swin-L?
